index.js

```diff
@@ -66,9 +66,9 @@ function Cabal (storage, key, opts) {
 }
 
 inherits(Cabal, events.EventEmitter)
-Cabal.prototype.getProtocolVersion = function (cb) {
+Cabal.prototype.getDatabaseVersion = function (cb) {
```

Comment:
Is the code that changes the `~/.cabal/archives` dir name already in master, or can that get into this PR too?

Reply:
@noffle it's not in master, but in the multi-cabal pr here https://github.com/cabal-club/cabal/pull/89

(i'm not sure how i would bring it in the cabal-core pr)

Reply:
Ah, of course!

My thinking was "gosh it'd be nice if we could get in the db-dir changes so we can test that separately from the multi-cabal work".